roles/custom/matrix-dendrite/defaults/main.yml

```diff
@@ -101,6 +101,9 @@ matrix_dendrite_enable_registration_captcha: false
 matrix_dendrite_recaptcha_public_key: ""
 matrix_dendrite_recaptcha_private_key: ""
 matrix_dendrite_recaptcha_siteverify_api: "https://www.google.com/recaptcha/api/siteverify"
```

Comment:
Maybe `matrix_dendrite_recaptcha_siteverify_api` should be defaulted to `''` to match the examples here: https://github.com/matrix-org/dendrite/blob/31f56ac3f43b692a56718fd1acfcb5a46e485c57/dendrite-sample.monolith.yaml#L183-L184 which say "keep it empty for reCAPTCHA".

Reply:
I see, thanks.